openlibrary/macros/WorkCarousel.html

```diff
@@ -0,0 +1,5 @@
+$def with(work)
+
+$if work:
+    $:render_template("home/custom_ia_carousel", title="You might also like:", key="related-subjects-carousel", work_id=work.key, _type="subjects", limit=42)
```

Comment:
Can we drop the ':'? The face that it's bold/big text already communicates that it's a heading, and the colon is a little inconsistent with the carousels on the homepage.